assets/stylesheets/application.scss

```diff
@@ -4,9 +4,8 @@ $govuk-assets-path: "/static/govuk/assets/";
 $govuk-new-link-styles: true;
 $govuk-link-underline-offset: .2em;
 
-// // import all govuk frontend scss
-// ideally we would remove this dependence on frontend
-@import "node_modules/digital-land-frontend/digital-land-frontend/dl-frontend";
+// import all govuk frontend scss
+@import "node_modules/govuk-frontend/govuk/all";
```

Comment:
previously, govuk scss was imported via dl-frontend. we should just directly import it instead